src/modules/analysis/AnalysisAddComponent.jsx

```diff
@@ -55,29 +88,42 @@ function newLine() {
   </Form>;
 }
 
+const getVal = (form, field) => form[field] && (form[field].rawValue || '');
```

Comment:
its better to have explicit call to form like for ex form.note than having it referenced indirectly with two params form and 'note'